drift/lib/src/runtime/types/mapping.dart

```diff
@@ -81,6 +81,15 @@ final class SqlTypes {
       return dartValue.rawSqlValue;
     }
 
+    if (dartValue is GeopolyPolygon) {
```

Comment:
There are a bunch of cases where I didn't do a good job at that, but I'm trying to move the core `package:drift/drift.dart` library to be more agnostic of the underlying DBMS. So supporting geopoly (a sqlite3-only module) here makes it more complicated to make drift support more databases in the future.

I fully support the analysis additions around geopoly types (because these are always opt-in), but I wonder if we should instead implement the runtime support for polygons in a separate library. It can still be part of the drift package, but perhaps under a different import (say `package:drift/extensions/geopoly.dart`, next to things like json1 support). We can then implement geopoly support with [custom types](https://drift.simonbinder.eu/docs/sql-api/types/) (albeit a custom type implemented in the drift package), which also makes it easier to define tables using polygon columns in Dart.
The generator should still apply that custom type by default if return values from the geopoly extension are involved, but that way we can let the core runtime focus on types supported by all SQL implementations.